### PR TITLE
Fix/tao 7833/update property labels

### PR DESCRIPTION
**Pull request by ferenckiss89**
Jira: https://oat-sa.atlassian.net/browse/TAO-7833

- JS check on regular property labels to keep them up to date
- unit test for the new method

### CODE REVIEW

File: views/js/form/post-render-props.js [hunk 1]
```diff
@@ -380,6 +381,23 @@ define([
...
+     * Checks and updates property labels 
+     * @param {Object} $properties - properties object
+     */
+    function _checkRegularPropertyLabels($properties) {
+        $properties.each(function() {
+            if($(this).hasClass('regular-property')) {
+                var $parentHeadingLabel = $(this).find('.property-heading-label');
+                var $editBlockLabel = $(this).find('.property-edit-container input[name$="_label"]');
+
+                if ($parentHeadingLabel.text() !== $editBlockLabel.val()) {
```
Comment (oatymart) [inline views/js/form/post-render-props.js]: I would say no need to even check for inequality. Just set it.

Comment (ferenckiss89) [reply views/js/form/post-render-props.js]: true

File: views/js/form/post-render-props.js [hunk 2]
```diff
@@ -380,6 +381,23 @@ define([
...
+     * Checks and updates property labels 
+     * @param {Object} $properties - properties object
+     */
+    function _checkRegularPropertyLabels($properties) {
+        $properties.each(function() {
+            if($(this).hasClass('regular-property')) {
+                var $parentHeadingLabel = $(this).find('.property-heading-label');
+                var $editBlockLabel = $(this).find('.property-edit-container input[name$="_label"]');
+
+                if (editBlockLabel.val() !== '') {
```
Comment (oatymart) [inline views/js/form/post-render-props.js]: $editBlockLabel

Comment (ferenckiss89) [reply views/js/form/post-render-props.js]: yep

**Pull request merged**